reactive_controller:
  forward_term: 0.2
  gain_tx: 0.00008
  gain_ty: 0.00008
  gain_tz: 0.00008
  gain_yaw: 0.00008
